vbench/src/tests/handler_thread/handler_thread_test.cpp

```diff
@@ -15,9 +15,11 @@ struct MyHandler : Handler<int> {
 
 MyHandler::~MyHandler() = default;
 
+VESPA_THREAD_STACK_TAG(test_executor);
```

Comment:
Consider _thread instead of _executor as this is used for HandlerThread.

Reply:
Fixed